Ladybird/HelperProcess.cpp

```diff
@@ -69,10 +70,14 @@ ErrorOr<NonnullRefPtr<WebView::WebContentClient>> launch_web_content_process(
                 arguments.append("--mach-server-name"sv);
                 arguments.append(server.value());
             }
-            Vector<ByteString> certificate_args;
-            for (auto const& certificate : web_content_options.certificates) {
-                certificate_args.append(ByteString::formatted("--certificate={}", certificate));
-                arguments.append(certificate_args.last().view());
+            Vector<String> fd_strings;
```

Comment:
:thonk: what's `fd_strings` for?

Reply:
oh, `arguments` holds StringViews. awkward but makes sense

Reply:
Yeah, had some nice UB debugging figuring that one out. Funny part is I already had it in place for certificates